src/js/d3ViewComponent.js

```diff
@@ -17,6 +17,8 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 1. Convert jQuery DOM elements to D3 elements;
     // 2. Attach D3 DOM event listeners;
     // 3. Synthesize that.options.styles and that.options.selectors to combine elements with the same key into that.classes
+    // 4. Create a basic SVG drawing area for use by implementing grades that actually
+    // draw charts
 
     fluid.defaults("floe.d3ViewComponent", {
```

Comment:
This is a good suggestion and an easy refactor to make `floe.svgDrawingArea` as a middle grade between `d3ViewComponent` and the actual chart-drawing grades - implementing.